### ISSUE TITLE
I am having issues determining if Autobot uses Fabric or Forge, or both?

### ISSUE THREAD

**Issue by darkstar01** [christopherthompson81/autobot#3]
I am having issues determining if Autobot uses Fabric or Forge, or both?

I am working on making a public Anarchy server where the hacks are provided for your pleasure.

Sorry for dumb questions, the only reason I even know about this mod is because of FitMC.

**Comment by darkstar01** [comment]
Also, how do I compile this?

**Comment by TheCrazyInsanity** [comment]
This isnt the autobot your looking for, its here: https://github.com/bebeli555/AutoBot
